tests/package-title-dir-rtl-001/OPS/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+        <dc:coverage>Internationalization</dc:coverage>
+        <dc:creator>Ivan Herman</dc:creator>
+        <dc:date>2021-09-22</dc:date>
```

Comment:
Opinion: should we have real dates here? I kinda like them, but we do not have real dates in other tests. Everything else is 2021-01-01.

Reply:
Dates are currently not used, that is true, but I myself would prefer to use real dates when possible. It may help us, later, to trace back when the tests have been done...

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357